.github/workflows/ci-build.yml

```diff
@@ -108,16 +90,12 @@ jobs:
           - test-apps/base-tests
         ember-try-scenario:
```

Comment:
to cut down on CI noise, this is now just testing the ranges of support,

3.16 - minimum
x.latest - last LTS for a major, X
the 3 variable embers

Reply:
@NullVoxPopuli looks like some kind of mistake as now it only has 4.0, 4.12 and 5.12, no more 3.16 and no 3.28?

Reply:
this comment is out of date actually -- we cannot support ember < v4